WordPress/UITestsFoundation/Globals.swift

```diff
@@ -53,6 +53,22 @@ public func waitAndTap( _ element: XCUIElement, maxRetries: Int = 10) {
     }
 }
 
+public func waitForElementToDisappear( _ element: XCUIElement, maxRetries: Int = 10) {
+    var retries = 0
+    while retries < maxRetries {
+        if element.isHittable {
```

Comment:
Have you considered using `element.exists` instead of `element.isHittable`?  

It could make the function more generic as an element can become not hittable without necessarily disappearing.

Reply:
That's a good point! I'll update to use `exists`